src/Resources/config/doctrine/AbstractFragment.orm.xml

```diff
@@ -3,7 +3,7 @@
     <mapped-superclass name="Sonata\ArticleBundle\Entity\AbstractFragment">
         <field name="backofficeTitle" type="string" column="backoffice_title" length="255"/>
         <field name="position" type="integer" column="position" nullable="true"/>
-        <field name="fields" type="json" column="settings"/>
+        <field name="fields" type="json" column="fields"/>
```

Comment:
This is a BC break. A BC way to do it would be to convert the mapping into php, and have an option to to use the correct name, and trigger a deprecation if the option is not set to true

Reply:
Indeed it's a bc break, but we can put this on a new 1.2.0 version no ?
Less changes and since this bundle it's pretty new, it won't impact a lot of peoples except ekino.

Reply:
Maybe it should have been 0.x, but https://semver.org/ says it cannot happen on a minor version. Database BC-breaks are the biggest kind of BC-breaks IMO, this would not send a good signal to consumers. And we have no guarantee everyone here is ekino: https://packagist.org/packages/sonata-project/article-bundle/stats

Reply:
Here are some docs about what I told you: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/php-mapping.html#php-mapping